.github/workflows/update-geoip.yml

```diff
@@ -24,14 +16,25 @@ concurrency:
 jobs:
   certificate-check:
     name: "Check for updated GeoIP DB"
+
     # Don't run the cron job on forks.
-    if: ${{ github.event_name != 'schedule' || github.repository == 'YOURLS/YOURLS' }}
+    # Don't run on a branch that was created by the workflow
+    if: |
+      ${{ github.event_name == 'schedule' && github.repository != 'YOURLS/YOURLS' }} &&
+      ${{ github.ref != 'refs/heads/auto-update-geoip' }}
 
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Generate token
+        uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.TRIGGERTRIGGER_APP_ID }}
+          private_key: ${{ secrets.TRIGGERTRIGGER_PRIVATE_KEY }}
```

Comment:
Are this secrets registered at repo level or at org level?
Can I rename the app to "YOURLS Bot" in order to use it on other repos as well?

Reply:
I put it at repo level but sure it can be renamed and moved. Is this even possible, not sure

Reply:
Done 👍